tests/test_auction.py

```diff
@@ -51,4 +51,22 @@ def test_admin_nominate_from_nonadmin_raises(started_auction):
                 author=mock.Mock(id=0),
             )
         )
-        print(e)
\ No newline at end of file
+        print(e)
+
+def test_add_player_from_command(started_auction):
+    with pytest.raises(AuctionValidationError) as e:
+        started_auction.player(
+            message=mock.Mock(
+                content="$player test 35",
+                author=mock.Mock(id=0), 
+            )
+        )
+
+def test_add_captain_from_command(started_auction):
+    with pytest.raises(AuctionValidationError) as e:
+        started_auction.captain(
+            message=mock.Mock(
+                content="$captain test 1000",
+                author=mock.Mock(id=0), 
+            )
+        )
```

Comment:
i don't think these tests test anything. the author has ID 0, so isn't an admin and will explode, and the `pytest.raises` block is basically asserting that this stuff explodes. kill the `raises` stuff, pass in an admin ID, and then after you add the captain/player you should assert something about "are they actually in the db/captainlist/playerlist"